recipe/patches/0018-Fix-flags-for-un-vendoring-abseil-cpp.patch

```diff
@@ -23,19 +23,10 @@ index c8b4716b124..5d4a86886c1 100644
 +            #    absl::raw_hash_set
 +            #    absl::city absl::hash absl::wyhash
 +            #    absl::numeric)
-+            'absl_raw_hash_set', 'absl_hash', 'absl_wyhash',
-+            'absl_hashtablez_sampler', 'absl_exponential_biased',
-+            'absl_synchronization', 'absl_graphcycles_internal',
-+            'absl_stacktrace', 'absl_symbolize', 'absl_malloc_internal',
-+            'absl_debugging_internal', 'absl_demangle_internal', 'absl_time',
-+            'absl_civil_time', 'absl_time_zone', 'absl_city',
-+            'absl_bad_optional_access', 'absl_strings', 'absl_throw_delegate',
-+            'absl_strings_internal', 'absl_base', 'absl_spinlock_wait',
-+            'absl_bad_variant_access', 'absl_raw_logging_internal',
-+            'absl_log_severity', 'absl_int128'
++            'absl_dll'
```

Comment:
Confusingly (or infuriatingly), this is actually called `abseil_dll`

Reply:
and it only gets triggered at the way end of the build ^_^

Reply:
I think this patch should be reverted for unix, and updated only for windows.

Reply:
That was kinda my point 🙃